package.json

```diff
@@ -11,7 +11,24 @@
     "watermark": "./scripts/watermark.sh",
     "dev": "node server.js"
   },
+  "lint-staged": {
+    "*.css": [
+      "stylelint --fix",
+      "git add"
+    ]
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
   "dependencies": {
     "express": "^4.17.1"
+  },
+  "devDependencies": {
+    "husky": "^3.1.0",
+    "lint-staged": "^9.5.0",
```

Comment:
Oh, I see, this is something lint-staged knows how to do! https://www.npmjs.com/package/lint-staged